### Is voter turnout influenced by the weather?  

Let pctvote be the percent of the registered voters in a given county who turn out to vote (i.e., pctvote = 80 means that 80% of registered voters in a given county turned out to vote).  

Let *rain* be a **dummy** variable for whether it rains for at least one hour on election day,  *temp* be the high temperature on election day (in degrees), and *numpoll* be the number of polling places in the county.  

One model to test the relationship is 

$\textrm{pctvote}=\beta_0+\beta_1\textrm{rain}+\beta_2\textrm{temp}+\beta_3\textrm{numpoll}+u$

### Null Hypotheses &  Alternative Hypothesis
$$H0: {\beta}_1 =0$$
$$H1: {\beta}_1 \not=0$$

### What I expect from ${\beta}_1$?

${\beta1}$ represents the percent change in pctvote (percent change of registered voters who turn out to vote) associated with one unit increase in rain, while holding all other variables constant. Based on the situation, it is likely that less rain would result in higher voter turnout because people are more willing to go out when they don't risk getting wet. Conversely, heavy rain compared to light rain would likely discourage people from driving to the polling stations or dropping off their ballots. Therefore, if one unit of rain is associated with ${\beta1}$, I expect ${\beta1}$ to be a negative value, indicating that increased rainfall leads to a decrease in voter turnout.


### Suppose we estimate the model using data on 120 randomly selected counties in the U.S., and we deliver the estimates (with standard errors in parentheses) below.

$$\hat{\beta}_0=41.67 (1.262)$$
$$\hat{\beta}_1=-4.92 (0.799)$$
$$\hat{\beta}_2=0.01 (0.017)$$
$$\hat{\beta}_3=0.65 (0.027)$$
$$R^2=0.398$$



### Test the hypothesis in part 1 at the 1% significance level.  Be sure to state the critical value that you use.

In [9]:
t_stat = (-4.92 -0 )/0.799
critical_value <- -2.33

t_stat
critical_value

[1] -6.157697

[1] -2.33

The test statistic value is less than the critical value (-6.16 < -2.33) therefore we can reject the null. We can not claim that the percent change is 0 per unit of rain.